src/main/java/Bob.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Scanner;
+
+public class Bob {
+
+    public static final String SEPARATOR = "____________________________________________________________";
+
+    public static void main(String[] args) {
+        String logo = "  ____        _     \n"
+                + " | |_) \\ ___ | |___  \n"
+                + " |  _ //  _  \\   _ \\ \n"
+                + " | |_)\\\\ (_) /  |_) |\n"
+                + " |____/ \\___/|_|___/ \n";
+        System.out.println(logo);
+        System.out.println("Hello! I'm Bob");
+        System.out.println("What can I do for you?");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        while (true) {
+            String input = scanner.nextLine();
+            if (input.equals("bye")) {
+                System.out.println(SEPARATOR);
+                System.out.println("Bye. Hope to see you again soon!");
+                System.out.println(SEPARATOR);
+                break;
+            } else if (input.equals("list")) {
+                System.out.println(SEPARATOR);
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskCount; i++) {
+                    System.out.println((i+1) + "." + tasks[i]);
+                }
+                System.out.println(SEPARATOR);
+            } else if (input.startsWith("mark ")) {
+                String[] inputsInString = input.split(" ");
+                int taskIndex = Integer.parseInt(inputsInString[1]) - 1;
+                tasks[taskIndex].markAsDone();
+                System.out.println(SEPARATOR);
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println("  " + tasks[taskIndex]);
+                System.out.println(SEPARATOR);
+            } else if (input.startsWith("unmark ")) {
```

Comment:
I like how you used startsWith to check for the command! Consider making them as constants as well?